### PR TITLE
feat: enable building Go projects without Node.js dependency

### PR DESCRIPTION
**Pull request by odsod**
Provides alternative to Prettier via markdownfmt

### CODE REVIEW

File: targets/mgmarkdownfmt/targets.go [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+	"path/filepath"
+
+	"github.com/go-logr/logr"
+	"github.com/magefile/mage/mg"
+	"github.com/magefile/mage/sh"
+	"go.einride.tech/mage-tools/mglog"
+	"go.einride.tech/mage-tools/mgtool"
+)
+
+const version = "75134924a9fd3335f76a9709314c5f5cef4d6ddc"
```
Comment (viktorvoltaire) [inline targets/mgmarkdownfmt/targets.go:16]: Why are we depending on a commit rather then a version?

Comment (odsod) [reply targets/mgmarkdownfmt/targets.go:16]: No recent version has been released

File: targets/mgmarkdownfmt/targets.go [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+}
+
+func prepare() error {
+	const toolName = "markdownfmt"
+	toolDir := filepath.Join(mgtool.GetPath(), toolName, version)
+	executable = filepath.Join(toolDir, toolName)
+	if _, err := os.Stat(executable); err == nil {
+		return nil
+	}
+	return sh.RunWithV(
```
Comment (viktorvoltaire) [inline targets/mgmarkdownfmt/targets.go:34]: Please use https://github.com/einride/mage-tools/blob/b17068b60126a0460b2c31a1dafa1745a9c2c37d/mgtool/build.go#L13 `GoInstall` instead :) here is an example on when we use it https://github.com/einride/mage-tools/blob/main/targets/mggolangmigrate/targets.go#L25

Comment (odsod) [reply targets/mgmarkdownfmt/targets.go]: Fixed!

**Review by viktorvoltaire — APPROVED** [review]
👌

**Pull request merged**